sdk_container/src/third_party/coreos-overlay/x11-drivers/nvidia-drivers/nvidia-drivers-535.230.02.ebuild

```
@@ -23,6 +23,8 @@ src_install() {
   exeinto "/usr/lib/nvidia/bin"
   doexe "${FILESDIR}/bin/install-nvidia"
   doexe "${FILESDIR}/bin/setup-nvidia"
+  cp "${FILESDIR}/nvidia-metadata" nvidia-metadata || die "cp failed"
+  sed -i -e "s/@PV@/${PV}/" nvidia-metadata
```

Better to use the temporary directory ${T}:

```suggestion
  sed -e "s/@PV@/${PV}/" "${FILESDIR}/nvidia-metadata" >"${T}/nvidia-metadata" || die "sed failed"
```

---

done